Makefile

```diff
@@ -38,3 +45,16 @@ build-server: swig
 
 clean: clean-sdk
 	rm -rf bin gen/gen_wrap.cxx gen/gen.go
+
+appimage: build-module
+	cd etc/packaging/appimages && appimage-builder --recipe rplidar-module-`uname -m`.yml
+	mkdir -p etc/packaging/appimages/deploy/
+	mv etc/packaging/appimages/*.AppImage* etc/packaging/appimages/deploy/
+	chmod a+rx etc/packaging/appimages/deploy/*.AppImage
+
+clean-appimage:
+	rm -rf etc/packaging/appimages/AppDir
+	rm -rf etc/packaging/appimages/appimage-build
+	rm -rf etc/packaging/appimages/deploy
```

Comment:
These should probably all be on line/command. Less chance of an error on one and then it exiting before cleaning the others.